File: backend/services/post-ts-koa-mongo-fdm/src/common/api/invalid-password.ts
import { ErrorBody } from "../utils/error";

export const empty: ErrorBody = {
    error: {
        error: "PASSWORD_EMPTY",
        reason: "Password cannot be empty",
        message: "Password cannot be empty",
    }
};

export const tooShort: ErrorBody = {
    error: {
        error: "PASSWORD_TOO_SHORT",
        reason: "password is too short",
        message: "password is too short",
    },
};

export const tooLong: ErrorBody = {
    error: {
        error: "PASSWORD_TOO_LONG",
        reason: "password is too long",
        message: "password is too long",
    },
};

File: backend/services/post-python-fastapi-mongo-fdm/src/api/invalid_password.py
from typing import Final
from utils.error import ErrorBody

empty: Final[ErrorBody] = {
    "error": {
        "error": "PASSWORD_EMPTY",
        "reason": "Password cannot be empty",
        "message": "Password cannot be empty",
    }
}

tooShort: Final[ErrorBody] = {
    "error": {
        "error": "PASSWORD_TOO_SHORT",
        "reason": "Password is too short",
        "message": "Password is too short",
    }
}

tooLong: Final[ErrorBody] = {
    "error": {
        "error": "PASSWORD_TOO_LONG",
        "reason": "Password is too long",
        "message": "Password is too long",
    }
}
